### DISCUSSION TITLE
Change car's individual network

### DISCUSSION THREAD

**Discussion by JaneClelandHuang** [prl-mushr/mushr#47] [Q&A]
Is there an easy way to reset each car's individual network to Robot1, Robot2, ....., RobotN?  If not, we'll move to semi-static IP addresses (but our university sometimes changes the IP address on us).

**Comment by palanc** [comment]
Hi @JaneClelandHuang - I think the easiest way would be to open the Connection Editor on each of the cars and change the network name. In particular, when you open the Connection Editor GUI you should see something similar to the figure shown under **step 8** here: [https://mushr.io/hardware/nano_install/](https://mushr.io/hardware/nano_install/)
There you can change both the 'Connection name' and 'SSID' field to RobotN, then save, then either restart networking or just reboot the robot. 
That's very exciting for us to hear that you have N robots (i.e. more than one) :)

**Reply by JaneClelandHuang** [reply]
Great - thank you!  I missed that bit and it looks quite straightforward.  We've built 10 of them and I'm using them in my Software Engineering for Unmanned Autonomous Vehicle course.
